src/isomorphic/__tests__/React-test.js

```diff
@@ -35,10 +35,11 @@ describe('React', () => {
     expect(createClass).not.toBe(undefined);
     expectDev(console.error.calls.count()).toBe(1);
     expectDev(console.error.calls.argsFor(0)[0]).toContain(
-      'React.createClass is no longer supported. Use a plain ' +
-        "JavaScript class instead. If you're not yet ready to migrate, " +
+      'React.createClass is no longer supported. Use a plain JavaScript ' +
+        "class instead. If you're not yet ready to migrate, " +
         'create-react-class is available on npm as a temporary, ' +
-        'drop-in replacement.',
+        'drop-in replacement. ' +
```

Comment:
Should we make it clear it will keep on working indefinitely? "temporary" doesn't sound very reassuring.